dev/public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>BPMN Visualization Demo</title>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.1.1/css/all.css">
```

Comment:
Here we are still loading the whole fontawesome resources whereas we are only using 2 classes (the 2 icons of the upload BPMN diagram button).
We may use https://fontawesome.com/v6/docs/web/setup/use-kit to create a minimal file that fit our needs.